racket/src/zuo/zuo-doc/overview.scrbl

```diff
@@ -307,7 +307,7 @@ specific choices about how to work with paths:
        /home/racket/scripts/go.zuo}, then @racket[quote-module-path]
        reports an absolute path. Similarly, with
        @racketmodname[zuo/build], when you use a relative path to
-       refer to a dependency, then information about the dependency
```

Comment:
IMO, I would rather have a comma here. Without a comma, I read "dependency information" as a noun, and it's very confusing.

Reply:
You're right. I'll amend the commit